integration-tests/src/test/java/com/redhat/service/bridge/integration/tests/steps/ProcessorSteps.java

```diff
@@ -49,7 +70,29 @@ public void addProcessorToBridgeWithBody(String testBridgeName, String processor
         assertThat(response.getKind()).isEqualTo("Processor");
         assertThat(response.getHref()).isNotNull();
         assertThat(response.getStatus()).isEqualTo(BridgeStatus.ACCEPTED);
-        assertThat(response.getFilters().size()).isEqualTo(filtersSize);
+    }
+
+    @Then("add a Processor to the Bridge \"([^\"]*)\" with body is failing with HTTP response code (\\d+):$")
+    public void newProcessorIsAddedToBridgeWithBodyIsFailingWithHTTPResponseCode(String testBridgeName,
+            int responseCode, String processorRequestJson) {
+        BridgeContext bridgeContext = context.getBridge(testBridgeName);
+        InputStream resourceStream = new ByteArrayInputStream(processorRequestJson.getBytes(StandardCharsets.UTF_8));
```

Comment:
Could we please close this stream once not needed? I believe using try with resources could solve it easily.

Reply:
updated

Reply:
See https://github.com/5733d9e2be6485d52ffa08870cabdee0/sandbox/pull/394/commits/f232b5a4c4021862eebe9af536beed3ddabf6fe0